lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -491,31 +520,47 @@ export class UserAgentApplication {
         scopes = this.filterScopes(scopes);
       }
 
-      // Extract ADAL id_token if it exists
-      let idTokenObject;
-      idTokenObject = this.extractADALIdToken();
+      // construct extraQueryParams string from the request
+      let extraQueryParameters = Utils.constructExtraQueryParametersString(request.extraQueryParameters);
```

Comment:
why not do in one line, if not one line maybe make it clearer 

const extraQueryParams = ...
const extraQueryParamsWithPrompt = ..

Reply:
This code changed.